documentation/staging/content/userguide/managing-domains/domain-events.md

```diff
@@ -22,15 +22,24 @@ This document describes Kubernetes events that the operator generates about reso
 
 The operator generates these event types in a domain namespace, which indicate the following:
 
- *  `DomainCreated`: A new domain is created.
- *  `DomainChanged`: A change has been made to an existing domain.
- *  `DomainDeleted`: An existing domain has been deleted.
- *  `DomainProcessingStarting`: The operator has started to process a new domain or to update an existing domain. This event may be a result of a `DomainCreate`, `DomainChanged`, or `DomainDeleted` event, or a result of a retry after a failed attempt.
- *  `DomainProcessingFailed`: The operator has encountered a problem while it was processing the domain resource. The failure either could be a configuration error or a Kubernetes API error.
- *  `DomainProcessingRetrying`: The operator is going to retry the processing of a domain after it encountered a failure.
- *  `DomainProcessingCompleted`:  The operator successfully completed the processing of a domain resource.
- *  `DomainProcessingAborted`:  The operator stopped processing a domain when the operator encountered a fatal error or a failure that persisted after the specified maximum number of retries.
- *  `DomainRollStarting`:  The operator has detected domain resource or Model in Image model
+ * `Created`: A new domain is created.
+ * `Changed`: A change has been made to an existing domain.
+ * `Deleted`: An existing domain has been deleted.
+ * `Available`: An existing domain has become available, which means a sufficient number of servers are ready such that the customer's applications are available.
+ * `Failed`: The domain resource encountered a problem which prevented it to become fully up. The possible failure could be one or multiple of the following conditions:
+   * Invalid configurations in domain resource;
```

Comment:
domain resource; -> domain resource. (Please put a period, not a semi-colon, at the end of each bulleted item.)